react/src/views/Transfers/ShipmentView/components/ShipmentTabs.tsx

```diff
@@ -35,7 +35,7 @@ function ShipmentTabs({ shipmentDetail }: IShipmentTabsProps) {
       </TabList>
       <TabPanels>
         <TabPanel p={0}>
-          {(shipmentDetail.length || 0) === 0 && (
+          {(shipmentDetail?.length || 0) === 0 && (
```

Comment:
@vahidbazzaz please ask chatgpt to fix this or assure that shipmentDetail is not undefined before.